docker/coraza-spoa/docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
```

Comment:
Shall we add a comment on why this is needed?

Reply:
👍🏻